reconcile/quay_permissions.py

```diff
@@ -86,39 +86,38 @@ def run(dry_run):
                             continue
 
                         perm_org_key = (
-                          permission['quayOrg']['instance']['name'],
-                          permission['quayOrg']['name']
+                            permission['quayOrg']['instance']['name'],
+                            permission['quayOrg']['name']
                         )
 
                         if perm_org_key != org_key:
-                            logging.warning('wrong org, should be %s', org_key)
+                            logging.warning(
+                                f'wrong org, should be {org_key}'
+                            )
```

Comment:
There are endless discussions about avoiding interpolation in logging or not... In this case it should not matter but I'm all for avoiding unrelated changes in a PR 😂